plugins/inputs/ah_wireless/ah_wireless_defines.go

```diff
@@ -876,6 +885,7 @@ type ah_dcd_stats_report_rate_stats struct  {
 	kbps				uint32     /* TX/RX rate Kbps */
 	rate_dtn			uint8      /* TX/RX bit rate distribution */
 	rate_suc_dtn		uint8     /* TX/RX bit rate sucess distribution */
+	pad			[2]byte
```

Comment:
Padding not required since it is not used in ioctl

Reply:
Okay , I would revert this padding and test if rate's are getting properly updated.